### PR TITLE
Add ruby custom build command

### PR DESCRIPTION
**Pull request by william-msft**
- [x] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as https://devdiv.visualstudio.com/DevDiv/_boards/board/t/Oryx/Stories/?workitem=1576740
- [ ] Tests are included and/or updated for code changes.
- [ ] Proper license headers are included in each file.

### CODE REVIEW

**Comment by william-msft** [comment]
cc: @jochenjonc

**Review by jochenjonc — APPROVED** [review]

**Review by cormacpayne — APPROVED** [review]
Small comment about testing, otherwise LGTM :shipit:

File: src/BuildScriptGeneratorCli/Options/RubyScriptGeneratorOptionsSetup.cs [hunk 1]
```diff
@@ -23,6 +23,7 @@ public RubyScriptGeneratorOptionsSetup(IConfiguration configuration)
         public void Configure(RubyScriptGeneratorOptions options)
         {
             options.RubyVersion = this.GetStringValue(SettingsKeys.RubyVersion);
+            options.CustomBuildCommand = this.GetStringValue(SettingsKeys.CustomBuildCommand);
```
[the remark this answers is not in the document]
Comment (william-msft) [reply src/BuildScriptGeneratorCli/Options/RubyScriptGeneratorOptionsSetup.cs:26]: We do, but looks like we already had one: https://github.com/microsoft/Oryx/blob/2664d28fd1dd1b98e38097b99ceacb1d75eeba1c/tests/Oryx.BuildImage.Tests/Ruby/RubySampleAppsTest.cs#L152

**Comment by jochenjonc** [comment]
@william-msft any idea when this version of Oryx will land in the Azure/static-web-apps-deploy@v1 GitHub Action?

**Comment by william-msft** [comment]
> https://github.com/microsoft/Oryx/releases/tag/20220718.1

@anthonychu is there someone from SWA who would know when this could be released on SWA? The Oryx tag is above.

**Pull request merged**